Failing frontend build

I tried to build VP 3.0.2 but webpack build fails on following error:

```
/Users/janvoracek/Projects/versionpress/versionpress/frontend/config/webpack-build.js:14
    throw new gutil.PluginError('webpack', buildError);
    ^
Error: ./src/CommitPanel/CommitPanelOverview.react.tsx
(6,14): error TS2503: Cannot find namespace 'VpApi'.
[10:31:33] 'frontend-build' errored after 12 s
[10:31:33] Error in plugin 'gulp-shell'
Message:
    Command `npm run build` failed with exit code 1
[10:31:33] 'frontend-build-and-deploy' errored after 23 s
[10:31:33] Error in plugin 'run-sequence'
Message:
    An error occured in task 'frontend-build'.
```

`VpApi` was introduced in #1079.

--- a/frontend/src/services/WpApi.ts
+++ b/frontend/src/services/WpApi.ts
@@ -1,5 +1,5 @@
 /// <reference path='../../typings/typings.d.ts' />
-/// <reference path='./Wp.d.ts' />
+/// <reference path='./VpApi.d.ts' />
 
 import * as request from 'superagent';
 import config from '../config';

